tests/acceptance/Subscribers/SubscriberCookieCest.php

```diff
@@ -32,12 +32,12 @@ public function setSubscriberCookieOnSignup(AcceptanceTester $i) {
       $i->click('#wp-submit');
       $i->waitForText('Registration complete. Please check your email');
     } else {
-      $i->fillField(['name' => 'user_name'], 'mu-test-user');
+      $i->fillField(['name' => 'user_name'], 'muTestUser');
```

Comment:
Is this change needed for the new AWS locations? And why did you that?

Reply:
@lysyjan No, this is not needed for new AWS locations. This is more to fix a flaky CircleCI acceptance test failure.

Reply:
@triple0t Okay, I get it! Thank you for the explanation! Next time, could you please create a specific commit for a similar change?

Reply:
@lysyjan  Noted. Would you prefer I remove this change now or leave it as it is?

Reply:
@triple0t It looks that Pavel will create a PR with this change, and I guess it will probably be merged earlier. So, leave it as it is. Thank you

Reply:
@lysyjan @triple0t just to note, this change include uppercase, while Pavel's PR specifically says it can only contain lowercase

Reply:
See the artifact here: https://circle-production-customer-artifacts.s3.amazonaws.com/picard/56fbdc1ec9e77c000141d0d7/61dd7d0dc7a0aa116329f898-0-build/artifacts/tests/_output/MailPoet.Test.Acceptance.SubscriberCookieCest.setSubscriberCookieOnSignup.fail.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20220111T131022Z&X-Amz-SignedHeaders=host&X-Amz-Expires=60&X-Amz-Credential=AKIAJR3Q6CR467H7Z55A%2F20220111%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=21165e65f9b92f0b677299a7cd57d534c4ec94ab344ae2b97d2b2f8190248f21

Reply:
Thank you guys, I didn't know that.
@triple0t Please, can you remove this change from your commit? It would prevent a conflict, and also, a username cannot contain an upper case letter.